worlds/jakanddaxter/regs/BoggySwampRegions.py

```diff
@@ -0,0 +1,170 @@
+                         (state.has_all({"Roll", "Roll Jump"}, player)
+                          or state.has_all({"Double Jump", "Jump Kick"}, player)))
+    first_tether.connect(second_jump_pad)
+    first_tether.connect(first_pole_course)
+
+    first_tether_rat_colony.connect(first_tether)
+
+    second_jump_pad.connect(first_tether)
+
+    first_pole_course.connect(first_tether)
+    first_pole_course.connect(second_tether)
+
+    second_tether.connect(first_pole_course, rule=lambda state: can_jump_higher(state, player))
+    second_tether.connect(second_bats)
+
+    second_bats.connect(second_tether)
+    second_bats.connect(third_jump_pad)
+    second_bats.connect(fourth_jump_pad)
+    second_bats.connect(flut_flut_pad)
+
+    third_jump_pad.connect(second_bats)
+    fourth_jump_pad.connect(second_bats)
+
+    flut_flut_pad.connect(second_bats)
+    flut_flut_pad.connect(flut_flut_course, rule=lambda state: state.has("Flut Flut", player))  # Naturally.
+    flut_flut_pad.connect(farthy_snacks)
+
+    flut_flut_course.connect(flut_flut_pad)
+
+    farthy_snacks.connect(flut_flut_pad)
+    farthy_snacks.connect(box_field, rule=lambda state: can_jump_higher(state, player))
+
+    box_field.connect(farthy_snacks, rule=lambda state: can_jump_higher(state, player))
+    box_field.connect(last_tar_pit, rule=lambda state: can_jump_farther(state, player))
+
+    last_tar_pit.connect(box_field, rule=lambda state: can_jump_farther(state, player))
+    last_tar_pit.connect(fourth_tether, rule=lambda state: can_jump_farther(state, player))
+
+    fourth_tether.connect(last_tar_pit, rule=lambda state: can_jump_farther(state, player))
+    fourth_tether.connect(main_area)  # Fall down.
+
+    multiworld.regions.append(main_area)
+    multiworld.regions.append(first_bats)
+    multiworld.regions.append(first_jump_pad)
+    multiworld.regions.append(first_tether)
+    multiworld.regions.append(first_tether_rat_colony)
+    multiworld.regions.append(second_jump_pad)
+    multiworld.regions.append(first_pole_course)
+    multiworld.regions.append(second_tether)
+    multiworld.regions.append(second_bats)
+    multiworld.regions.append(third_jump_pad)
+    multiworld.regions.append(fourth_jump_pad)
+    multiworld.regions.append(flut_flut_pad)
+    multiworld.regions.append(flut_flut_course)
+    multiworld.regions.append(farthy_snacks)
+    multiworld.regions.append(box_field)
+    multiworld.regions.append(last_tar_pit)
+    multiworld.regions.append(fourth_tether)
+
+    # If Per-Level Orbsanity is enabled, build the special Orbsanity Region. This is a virtual region always
+    # accessible to Main Area. The Locations within are automatically checked when you collect enough orbs.
+    if options.enable_orbsanity == EnableOrbsanity.option_per_level:
+        orbs = JakAndDaxterRegion("Orbsanity", player, multiworld, level_name)
+
+        bundle_size = options.level_orbsanity_bundle_size.value
+        bundle_count = int(200 / bundle_size)
```

Comment:
Again (last comment about it), you could do this in the regions
```suggestion
        bundle_count = 200 // bundle_size
```